src/Microsoft.AspNetCore.Razor.Runtime/Runtime/TagHelpers/TagHelperDesignTimeDescriptorFactory.cs

```diff
@@ -102,18 +119,10 @@ private XmlDocumentationProvider GetXmlDocumentationProvider(Assembly assembly)
 
             var documentationProvider = _documentationProviderCache.GetOrAdd(cacheKey, valueFactory: _ =>
             {
-                var assemblyLocation = assembly.Location;
-
-                if (string.IsNullOrEmpty(assemblyLocation) && !string.IsNullOrEmpty(assembly.CodeBase))
```

Comment:
@Eilon and I discussed offline:

`After discussion offline to maintain consistency we decided to also remove CodeBase fallback in the net451 scenario. This will keep CoreCLR and net451 scenarios more consistent.`